10.9-libcxx/stable/main/finkinfo/languages/gcc12-13.0.info

```diff
@@ -0,0 +1,315 @@
+Info2: <<
+Package: gcc12
+Version: 12.3.0
+Revision: 1
+Type: gccver (12)
+Distribution: 13.0
```

Comment:
Perhaps also add 14.0 here?

Reply:
Fink's naming scheme only allows for a single Distribution in a %v-Dist.info named file afaict, so this needs additional info files (added in next commit) – but a less prolific solution would be much preferred!